manager/middlewares/rbac.go

```diff
@@ -32,6 +32,8 @@ var granularPerms = map[string]string{
 	"BaselineUpdateHandler":        "patch:template:write",
 	"BaselineDeleteHandler":        "patch:template:write",
 	"BaselineSystemsRemoveHandler": "patch:template:write",
+	"TemplateSystemsUpdateHandler": "patch:template:write",
+	"TemplateSystemsDeleteHandler": "patch:template:write",
```

Comment:
Should this be  ```content-sources:templates:write``` ?

https://github.com/RedHatInsights/rbac-config/blob/master/configs/stage/roles/content-sources.json#L61

Reply:
good point